src/MockToken.sol

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.17;
+
+import "forge-std/console.sol";
```

Comment:
I think this need to be in src for tests to import, but we will remove them eventually